sync/upstream.py

```diff
@@ -466,78 +466,60 @@ def pr_head(self):
     def pr_commits(self):
         pr_head = self.pr_head
         if not pr_head:
-            logger.error(
-                "Can't get PR commits as the ref head could not be found for %s" %
-                self.process_name
-            )
-            return
+            raise ValueError("Can't get PR commits as the ref head could not be found for %s" %
+                             self.process_name)
         else:
-            pr_head = self.git_wpt.commit(pr_head)
+            pr_head = sync_commit.WptCommit(self.git_wpt, pr_head)
 
-        pr = None
+        merge_base = []
```

Comment:
I don't know why this is a list since it's either 0 or 1 commits

Reply:
The merge_base method returns a list with either 0 and 1 commit, and since I use it twice I create an empty list here so that later on I can do a single `if len(merge_base) == 0` check instead of checking the length each call.

Reply:
OK, it's a bit strange, but I guess it makes sense.